__tests__/server/ssrServer.spec.js

```diff
@@ -122,13 +125,13 @@ describe('ssrServer', () => {
       disableRequestLogging: true,
     });
 
-    expect(register).toHaveBeenCalledTimes(13);
-    expect(register.mock.calls[0][0]).toEqual(fastifySensible);
-    expect(register.mock.calls[1][0]).toEqual(ensureCorrelationId);
-    expect(register.mock.calls[2][0]).toEqual(fastifyCookie);
-    expect(register.mock.calls[3][0]).toEqual(logging);
-    expect(register.mock.calls[4][0]).toEqual(fastifyMetrics);
-    expect(register.mock.calls[5]).toEqual([
+    expect(register).toHaveBeenCalledTimes(22);
+    expect(register.mock.calls[1][0]).toEqual(fastifySensible);
+    expect(register.mock.calls[2][0]).toEqual(ensureCorrelationId);
+    expect(register.mock.calls[3][0]).toEqual(fastifyCookie);
+    expect(register.mock.calls[4][0]).toEqual(logging);
+    expect(register.mock.calls[5][0]).toEqual(fastifyMetrics);
+    expect(register.mock.calls[6]).toEqual([
```

Comment:
we should revisit these tests. just bumping the calls number does not make it obvious what should or should not be happening.